themes/default-bootstrap/js/modules/loyalty/js/loyalty.js

```diff
@@ -25,9 +25,9 @@
 
 $(document).ready(function() {
 	$(document).on('change', '#our_price_display', function(e){
-		updateLoyaltyView(parseInt($('#our_price_display').text()));
+		updateLoyaltyView(priceWithDiscountsDisplay);
 	})
-	updateLoyaltyView(parseInt($('#our_price_display').text()));
+	updateLoyaltyView(priceWithDiscountsDisplay);

```

Comment:
Is that a global variable? By only seeing this piece of code we could think it will be undefined.

Reply:
Yes u can check :
 
http://www.chafoins.com/prestashop/chemisiers/2-chemisier.html#/11-couleur-noir/2-taille-m

Open the console i add a console.log

Noir M is a combination with discount